denops/skkeleton/deps/dictionary.ts

```diff
@@ -1,4 +1,4 @@
-export * as yaml from "jsr:@std/yaml@1.0.4";
+export * as yaml from "jsr:@std/yaml@1.0.5";
 export * as msgpack from "https://esm.sh/@msgpack/msgpack@3.0.0-beta2";
 export { default as jsonschema } from "https://esm.sh/jsonschema@1.4.1";
```

Comment:
I think we should migrate `esm.sh` to `npm:` import.

```suggestion
export * as msgpack from "npm:@msgpack/msgpack@3.0.0-beta2";
export { default as jsonschema } from "npm:jsonschema@1.4.1";
```

Reply:
npm importにしてないのは当時のDenoの実装に問題があったからなので置き換えられるなら置き換えてもよさそう